src/base/OpaqueId.hh

```diff
@@ -7,10 +7,8 @@
 //---------------------------------------------------------------------------//
 #pragma once
 
-#ifndef __CUDA_ARCH__
 #    include <cstddef>
```

Comment:
You may want to also change the indentation.

Reply:
Yep, noticed that it didn't get formatted correctly ... I just ran clang-format on the whole codebase to keep things simple.